templates/docs-section.html

```diff
@@ -76,6 +88,10 @@ <h1>
                 <span class="docs-page-subtitle">({{section.extra.subtitle}})</span>
             {% endif %}
         </h1>
+        <a class="example__github" href="https://github.com/bevyengine/bevy-website/tree/main/content/{{ section.relative_path }}">
```

Comment:
I don't think we need this, given that we already have this at the bottom of each page (which I think is a better place for it). I think putting it here (especially unconditionally) is distracting and unnecessary.